pwa/src/hooks/filterCount.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { useQuery } from "react-query";
+import APIService from "../apiService/apiService";
+import APIContext from "../apiService/apiContext";
+
+export const useFilterCount = () => {
+  const API: APIService | null = React.useContext(APIContext);
+
+  const getCategoryCount = () =>
+    useQuery<any, Error>(["CategoryCount"], () => API?.FilterCount.getCategoryCount(), {

```

Comment:
In the other code bases (I think) we always use the following syntax for the keys: `"category-count"`.